added SQUARE.X keyboard (#10066)
qmk/qmk_firmware: keyboards/inett_studio/sqx/hotswap/readme.md

> # SQUARE.X keyboard from the iNETT Studio

Reviewer: ```suggestion
# iNETT Studio SQUARE.X hotswap
```

> An 60% keybaord from the iNETT studio's square series

Reviewer: ```suggestion
A 60% keyboard from the iNETT Studio's SQUARE series.
```

> Keyboard Maintainer: [astro](https://github.com/yulei)  
> Hardware Supported: SQX Keyboard
> Hardware Availability: [GH GB](https://geekhack.org/index.php?topic=107062.0)

Reviewer: ```suggestion
* Keyboard Maintainer: [astro](https://github.com/yulei)  
* Hardware Supported: SQUARE.X hotswap
* Hardware Availability: [Geekhack Group Buy](https://geekhack.org/index.php?topic=107062.0)
```
